pkgs/servers/web-apps/plausible/package.json

```diff
@@ -1,57 +0,0 @@
-{
-  "repository": {},
-  "license": "MIT",
```

Comment:
In the last version of plausible, the package.json is missing version number, it was causing problems with yarn.
I've landed a PR upstream, but unfortunately just after the release. We might still need package.json until the next release (1.5.0 which won't be before another 6 months).

Reply:
mkYarnModules requires a version number in package.json, but yarnSetupHook does not

Reply:
I'm glad we are getting rid of that really hard to debug error.